airesources/Julia/hlt/constants.jl

```diff
@@ -0,0 +1,31 @@
+export ADDITIONAL_PRODUCTIVITY, BASE_PRODUCTIVITY, BASE_SHIP_HEALTH, 
+    DOCKED_SHIP_REGENERATION, DOCK_RADIUS, DOCK_TURNS, DRAG, EXPLOSION_RADIUS,
+    EXTRA_PLANETS, INFINITE_RESOURCES, MAX_ACCELERATION, MAX_SHIP_HEALTH, MAX_SPEED,
+    MAX_TURNS, PLANETS_PER_PLAYER, PRODUCTION_PER_SHIP, RESOURCES_PER_RADIUS,
+    SHIPS_PER_PLAYER, SHIP_RADIUS, SPAWN_RADIUS, WEAPON_COOLDOWN, WEAPON_DAMAGE, 
+    WEAPON_RADIUS
+
+# Constants
+const ADDITIONAL_PRODUCTIVITY = 6
+const BASE_PRODUCTIVITY = 6
+const BASE_SHIP_HEALTH = 255
+const DOCKED_SHIP_REGENERATION = 0
+const DOCK_RADIUS = 4.0
+const DOCK_TURNS = 5
+const DRAG = 7.0
+const EXPLOSION_RADIUS = 10.0
+const EXTRA_PLANETS = 4
+const INFINITE_RESOURCES = true
+const MAX_ACCELERATION = 7.0
+const MAX_SHIP_HEALTH = 255
+const MAX_SPEED = 7.0
```

Comment:
This is technically an integer, since speeds are sent as integers to the game engine. I would prefer for it to be "7".